### PR TITLE
update go version 1.12 in ci

### PR DESCRIPTION
**Pull request by hgsgtk**
Hi, thanks for your awesome testing library. 

Following Go version updating, I suggest updating Go version 1.12 in ci.

Best,
Kazuki

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: go
 go:
   - '1.10'
```
Comment (andreynering) [inline .travis.yml]: Can you remove this line? We just need to test the last 2 Go versions.

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ language: go
...
   - '1.11'
```
Comment (andreynering) [inline .travis.yml]: 1.11.x

File: .travis.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ language: go
...
+  - '1.12'
```
Comment (andreynering) [inline .travis.yml]: 1.12.x

**Review by andreynering — COMMENTED** [review]
Hi @hgsgtk

Thanks for your PR!

I just requested a few changes, if you don't mind.

**Comment by hgsgtk** [comment]
@andreynering Thanks for your review. I fixed.

https://github.com/go-testfixtures/testfixtures/pull/47/commits/39389d07f199b3909495be4c02adfa83ce9efb55

**Comment by andreynering** [comment]
@hgsgtk Thanks!

**Pull request merged**